package.json

```diff
@@ -27,13 +31,21 @@
     "chai-string": "1.5.0",
     "eslint": "^8.24.0",
     "mocha": "9.2.2",
-    "nock": "^13.2.9"
+    "nock": "^13.2.9",
+    "typescript": "^4.8.4"
   },
   "dependencies": {
     "@datadog/datadog-api-client": "^1.3.0",
     "debug": "^4.1.0"
   },
   "engines": {
     "node": ">=12.0.0"
-  }
+  },
+  "files": [
+    "README.md",
+    "LICENSE",
+    "index.js",
+    "lib/**",
+    "dist/**"
+  ]
```

Comment:
This includes fewer files than we used to, but that’s probably a good thing. The tests and lint configurations and so on all used to get published, and no longer do with this configuration.